examples/linear_model/plot_sparse_logistic_regression_20newsgroups.py

```diff
@@ -18,19 +18,23 @@
 input features would be to use univariate feature selection followed by a
 traditional (l2-penalised) logistic regression model.
 """
-import time
+import timeit
+import warnings
 
 import matplotlib.pyplot as plt
 import numpy as np
 
 from sklearn.datasets import fetch_20newsgroups_vectorized
 from sklearn.linear_model import LogisticRegression
 from sklearn.model_selection import train_test_split
+from sklearn.exceptions import ConvergenceWarning
 
 print(__doc__)
 # Author: Arthur Mensch
 
-t0 = time.clock()
+warnings.filterwarnings("ignore", category=ConvergenceWarning,
+                        module="sklearn")
```

Comment:
I would argue that it's better to show the convergence warning to user rather than hide it.